libs/dao/src/main/java/com/akto/dto/CollectionConditions/CollectionCondition.java

```diff
@@ -0,0 +1,45 @@
+package com.akto.dto.CollectionConditions;
+
+import java.util.Map;
+import java.util.Set;
+import org.bson.conversions.Bson;
+import com.akto.dto.ApiInfo;
+import com.akto.dto.ApiCollectionUsers.CollectionType;
+
+public abstract class CollectionCondition {
+    private Type type;
+    private Operator operator;
+
+    public CollectionCondition(Type type, Operator operator) {
```

Comment:
There is a similar class already - `TestingEndpoints`

Reply:
sure, will inherit that class over this one to reduce code.